blogchain/src/app/layout.tsx

```diff
@@ -1,36 +1,32 @@
-"use client";
-
+import { AppProviders } from "@/components/app-providers";
+import Footer from "@/components/footer";
 import Header from "@/components/header";
-import { MintbaseWalletContextProvider } from "@mintbase-js/react";
 import "@near-wallet-selector/modal-ui/styles.css";
-import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { Metadata } from "next";
 import { Inter } from "next/font/google";
 import "./globals.css";
-import Footer from "@/components/footer";
 
 const inter = Inter({ subsets: ["latin"] });
 
-const queryClient = new QueryClient();
+export const metadata: Metadata = {
+  title: `Blogchain`,
```

Comment:
you can download the image on readme from imgur, add on public here and add like

   openGraph: {
      images:['./thumbnail.png'],
    },